src/components/measure/MeasureDirective.js

```diff
@@ -43,7 +43,7 @@ goog.require('ga_measure_service');
                 update(evt.target);
               });
             });
-            newFeature.changed();
+            update(newFeature);
```

Comment:
probably timing issue. We bind to `change` event just before, then try to call it.